### PR TITLE
Add KubeCon24 NA SLC announcement blog

### PR DESCRIPTION
**Pull request by kami619**
Fixes https://github.com/keycloak/keycloak-web/issues/514

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]

**Comment by ryanemerson** [comment]
LGTM, thanks @kami619

File: blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+:title: Meet Keycloak at KubeCon Salt Lake City, Utah in Nov 2024
+:date: 2024-04-10
+:publish: true
+:author: Kamesh Akella
+
+We are thrilled to announce that Keycloak will be at KubeCon Salt Lake City, Utah in Nov 2024. There are several Keycloak specific sessions lined up during this Conference, and we will be hosting a Kiosk at the Project Pavilion at KubeCon NA 2024.
```
Comment (ahus1) [inline blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc:6]: I hear at least two people from Hitachi plan to join us at the Kiosk. So I'd like to keep it a bit more open. Maybe we can reach out to them to have them listed here as well. 

```suggestion
We are thrilled to announce that Keycloak will be at KubeCon Salt Lake City, Utah in Nov 2024. There are several Keycloak specific sessions lined up during this conference, and we will be hosting a Kiosk at the Project Pavilion at KubeCon.
```

File: blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+== What is KubeCon?
+
+Keycloak's presence in the previous KubeCons was a huge success,
+and we continue to have a lot of fun interacting with Keycloak enthusiasts, users, newcomers alike.
+KubeCon is a fast-growing Cloud Native tech conference expected to have up to 8,000 developers,
+architects, and technical leaders onsite as well as thousands of participants virtually.
+
+KubeCon Salt Lake City will be held from Nov. 12th, 2024 through Nov. 15th, 20245, with many of the co-located events happening on Tuesday, Nov 12th, 2024.
```
Comment (ahus1) [inline blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc:15]:
```suggestion
KubeCon Salt Lake City will be held from Nov. 12th, 2024 through Nov. 15th, 2024, with many of the co-located events happening on Tuesday, Nov 12th, 2024.
```

File: blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+== Keycloak community Meet & Greet at the Project Pavilion
+
+https://github.com/ryanemerson[Ryan Emerson], https://github.com/mabartos[Martin Bartos] and https://github.com/kami619[Kamesh Akella] will be at the Keycloak kiosk at the https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/project-engagement/#project-pavilion[Project Pavilion]. This is a great chance to meet people who use Keycloak, contribute to Keycloak, take our survey about new Keycloak features, and get some cool swag!
```
Comment (ahus1) [inline blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc:19]:
```suggestion
https://github.com/ryanemerson[Ryan Emerson], https://github.com/mabartos[Martin Bartos], https://github.com/kami619[Kamesh Akella] and other contributors will be at the Keycloak kiosk at the https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/project-engagement/#project-pavilion[Project Pavilion]. This is a great chance to meet people who use Keycloak, contribute to Keycloak, take our survey about new Keycloak features, and get some cool swag!
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Hi Kamesh, thank you for writing this post. See below for some tuning.

File: blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc (hunk 1 above)
Comment (kami619) [reply blog/2024/keycloak-kubeconf24-na-slc-announcement.adoc:6]: Good Idea @ahus1 I will reach out to them and find the names if possible.

**Comment by kami619** [comment]
@ahus1 I think this should now have all the changes necessary to get published, please give it another swirl, when you have the time. Thank you.

**Review by ahus1 — APPROVED** [review]

**Pull request merged**